include/quadgrid_cpp_imp.h

```diff
@@ -20,7 +20,7 @@ quadgrid_t<T>::begin_cell_sweep () const {
 template <class T>
 void
 quadgrid_t<T>::set_sizes (idx_t numrows, idx_t numcols,
-                          double hx, double hy) {
+			  double hx, double hy) {
```

Comment:
Perché cambiare l'indentazione qui? Stai attento a fare solo le modifiche necessarie ed ad includere solo modifiche fra loro collegate in ogni commit?